models/Document/Page.php

```diff
@@ -17,7 +17,7 @@
 namespace Pimcore\Model\Document;
 
 use Pimcore\Messenger\GeneratePagePreviewMessage;
-use Pimcore\Model\Redirect;
+use Pimcore\Bundle\SeoBundle\Model\Redirect;
```

Comment:
Same here: The doDelete will break if the bundle is not installed.
Maybe we can also add a listener for this?